client/me/concierge/book/info-step.js

```diff
@@ -88,10 +99,11 @@ class InfoStep extends Component {
 
 		return (
 			<div>
+				<IsRebrandCitiesSite onChange={ this.setRebrandCitiesValue } siteId={ site.ID } />
```

Comment:
You can think of the `IsRebrandCities` component as a form input like other controlled form inputs in React. It notifies when its value changes, and we set that in state like other form inputs.